thrift/src/test/java/com/intellij/plugins/thrift/completion/ThriftEditorCompletionTest.java

```diff
@@ -29,10 +29,4 @@ public void testExistingClosingBrace2(){
     myFixture.type('>');
     myFixture.checkResult("struct Foo {1: list<Foo><caret> list}");
   }
-
-  public void testExistingCurlyBrace(){
-    myFixture.configureByText(ThriftFileType.INSTANCE, "service Foo {<caret>}");
-    myFixture.type('}');
-    myFixture.checkResult("service Foo {}<caret>");
-  }
```

Comment:
Test succeeds in my local, but fails on Cirrus CI.
```bash
com.intellij.plugins.thrift.completion.ThriftEditorCompletionTest > testExistingCurlyBrace FAILED
    junit.framework.ComparisonFailure: TEXT expected:<service Foo {}[]> but was:<service Foo {}[}]>
```
https://cirrus-ci.com/task/5954460338880512

Reply:
Could you please revert the test? Let's figure out why it's failing in CI.

Reply:
Sure. I've just reverted it. Please check the test.